packages/components/src/combobox-control/style.scss

```diff
@@ -0,0 +1,70 @@
+.components-combobox-control {
+	color: $dark-gray-500;
+	position: relative;
+}
+
+.components-combobox-control__label {
+	display: block;
+	margin-bottom: 5px;
+}
+
+.components-combobox-control__button {
+	border: 1px solid $dark-gray-200;
+	border-radius: 4px;
+	color: $dark-gray-500;
+	display: inline-block;
+	min-height: 30px;
+	min-width: 130px;
+	position: relative;
+	text-align: left;
+
+	&:focus {
```

Comment:
Should this be nested under `&-input`? `components-combobox-control__button` shouldn't be focusable, should it?

Reply:
Are we actually doing the `&-` pattern now? I seem to recall us avoiding it intentionally so as to make the code more legible. @aduth I think is AFK but he's my go-to syntax standards guru.

Reply:
This applies to the button and input wrapper.

`&-` is fine for modifiers, it becomes an issue when composing element selectors because then you can't search for the whole string to find the rules for a given element.

Reply:
I think it is fine for modifiers, yes, or more specifically the `&.is-foo` (or `&:focus` and friends pseudo-classes, etc). But based on what I see in the code here, I think this falls under the same sort of problematic "you can't search for the whole string" you mention. I expect to be able to search for `components-combobox-control__button-input`, and would be unsuccessful in doing so with the code as proposed.